src/plugin/places/placesmenu.js

```diff
@@ -397,7 +397,7 @@ plugin.places.menu.visibleIfIsPlace = function(context) {
 plugin.places.menu.onSpatialEdit_ = function(event) {
   var features = os.ui.menu.spatial.getFeaturesFromContext(/** @type {Object} */ (event.getContext()));
   if (features.length === 1) {
-    var feature = features[0];
+    var feature = features[1];
```

Comment:
This was correct as is. If the array length is 1, there will only be an item at index 0.